distributor-widget/README.md

```diff
@@ -46,47 +76,80 @@ Mews.Distributor({
 ```
 
-### Advanced usage <a id="advanced-usage"></a>
+It's just an example, the automatic binding can attach click event listener to any HTML element.
 
-If you need a more specific setup for opening Distributor, or you want to call some API functions on a Distributor instance, you can provide a callback function as the second argument to the initialization call - the instance is provided as an argument to the callback.
+{% hint style="info" %}
+Closing of Distributor is provided in the overlay by default, no configuration needed from your side.
+{% endhint %}
 
-Very common example for this is using a custom start and end date selectors that are part of your website and then passing user’s selection to Distributor.
+## Done!
 
-```javascript
+This is all you need for the basic setup of Mews Distributor.
+
+### What you've done:
+- On page with this setup, loader script will prepare `Mews.Distributor`.
+- After the page loads, your code will call `Mews.Distributor`. This will initialize Distributor Widget, create (for now) hidden overlay and bind opening actions to selected HTML elements, like buttons.
+- When users click these HTML elements, Distributor Widget overlay will open, and they can book through it.
+- They can close it anytime and see your page again.
+
+## Optional: Control Distributor Widget
+
+If you want to have a more customized setup, or you want to call some API functions on a Distributor instance to control it, you can provide a callback function as the second argument to the initialization call. 
+
+This callback is later called asynchronously with an argument - Distributor instance. By calling methods on this instance you can control Distributor.
+
+Very common example of this is using a custom start and end date selectors that are part of your website and then passing user’s selection to Distributor.
+
+```html
 <!-- Example of setting custom dates. Useful if you have i.e. own calendars on website. -->
 <script>
-Mews.Distributor({
-    configurationIds: [
-        'Your Distributor configuration id'
-    ]
-},
-function (distributor) {
-    // you can call API functions on a distributor instance here
-    // set different start and end date
-    distributor.setStartDate(new Date(2018, 1, 1));
-    distributor.setEndDate(new Date(2018, 1, 3));
-});
+Mews.Distributor(
+    {
+        configurationIds: [
+            'Your Distributor configuration id'
+        ],
+        openElements: '.distributor-open'
+    },
+    function (distributor) {
+        // you can call API functions on a distributor instance here
+        // set different start and end date
+        distributor.setStartDate(new Date(2018, 1, 1));
+        distributor.setEndDate(new Date(2018, 1, 3));
+    }
+);
 </script>
 ```
 
-To see a list of all available API calls, please consult [API ](reference.md#api-reference)section.
-
-Closing of Distributor is provided in the overlay by default, so you don’t have to worry about that.
+{% hint style="info" %}
```

Comment:
Should work like this: https://docs.gitbook.com/editing-content/markdown#hints-and-callouts